### PR TITLE
Deploy OSP13

### PR DESCRIPTION
**Pull request by akrzos**
Add tasks related to deploying OSP13 to deploy-scalelab.yaml

### CODE REVIEW

File: vars/deploy.yml [hunk 1]
```diff
@@ -5,10 +5,19 @@
 rhos_release_rpm: "{{ lookup('env', 'OSP_RHOS_RELEASE_URL') }}"
 
 # OSP/OSPd versioning and build
-version: "{{ lookup('env', 'OSP_RHOS_VERSION')|default('11', true) }}"
-rhel_version: "{{ lookup('env', 'OSP_RHOS_RHEL_VERSION')|default(7.4, true) }}"
-rhos_release: "{{ lookup('env', 'OSP_RHOS_RELEASE_VERSION')|default('11-director', true) }}"
-build: "{{ lookup('env', 'OSP_RHOS_BUILD')|default('z4', true) }}"
+version: "{{ lookup('env', 'OSP_RHOS_VERSION')|default('13', true) }}"
+rhel_version: "{{ lookup('env', 'OSP_RHOS_RHEL_VERSION')|default('7.5', true) }}"
```
Comment (mbruzek) [inline vars/deploy.yml]: Is 7.5 strictly necessary? I noticed we had RHEL 7.5 on the undercloud but 7.4 on the hypervisors. I am +1 to move to the latest, I just want to know if this is required or recommended for OSP 13.

Comment (akrzos) [reply vars/deploy.yml]: So `OSP_RHOS_RHEL_VERSION` populates the rhos-release command to add specific repos for that version of RHEL.  I think rebuilding a machine in the scale lab is automatically installing RHEL 7.5 and thus we have the newer version of RHEL.  OSP13 requires RHEL 7.5 as I understood it, and OSP11 does not.

**Comment by akrzos** [comment]
> A few things that came to mind when reviewing this code was:
>Do we need to update the version of os-client-config on the overcloud (in roles/overcloud-deploy/tasks/main.yml)?

We should not have to update the version since it uses a virtualenv, however I have not tested ostag with OSP13 so it may not work.  If this turns out to be the case I would suggest we put together a short python script similar to ostag that interacts directly with the ironic api.

>In the undercloud-install role we do some special workarounds for OSP 11 (which look to be correctly using the when: version == '11') are there any similar workarounds needed for OSP 13?

OSP13 should spawn more than one keystone worker so we should not need that specific workaround, but we can add any workarounds that are identified after this p/r is merged imo.

> Are the storage templates the only ones that need to be different for 13? Are the compute and controller files OK for use in OSP13?

Good point I have not actually checked if these templates change as well. Let me see if I can find specific documentation on those templates or passing through nvmes with OSP13.  We might just need a cluster to try a few times with to get this right unfortunately.

>I saw no errors or blockers in the PR itself. These are just questions I have when looking over this change.

**Comment by akrzos** [comment]
Environment is available thus merging and testing.

**Pull request merged**